client/src/components/TabNavigator/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import EntypoIcon from 'react-native-vector-icons/dist/Entypo';
+import FontAwesome5Icon from 'react-native-vector-icons/dist/FontAwesome5';
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import {styles} from './styles';
+
+// Chat, Contacts and Profile Components for testing the navigation and will be removed later
+function Chat() {
+  return (
+    <View style={{flex: 1, justifyContent: 'center', alignItems: 'center'}}>
+      <Text>Chat!</Text>
+    </View>
+  );
+}
+
+function Contacts() {
+  return (
+    <View style={{flex: 1, justifyContent: 'center', alignItems: 'center'}}>
+      <Text>Contacts!</Text>
+    </View>
+  );
+}
+
+function Profile() {
+  return (
+    <View style={{flex: 1, justifyContent: 'center', alignItems: 'center'}}>
+      <Text>Profile!</Text>
+    </View>
+  );
+}
+
+// to create the tab navigator
+const Tab = createBottomTabNavigator();
+
+function TabNavigator() {
+  return (
+    <Tab.Navigator
+      initialRouteName="Chat"
+      tabBarOptions={{
+        activeTintColor: '#51984C',
+        style: {...styles.tabNavigator},
+        labelStyle: {...styles.labelStyle},
+      }}>
+      <Tab.Screen
+        name="Chat"
+        component={Chat}
+        options={{
+          tabBarLabel: 'Chat',
+          tabBarIcon: ({color}) => (
+            <EntypoIcon name="chat" color={color} size={33} />
+          ),
+        }}
+      />
+
+      <Tab.Screen
+        name="Contacts"
+        component={Contacts}
+        options={{
+          tabBarLabel: 'Contacts',
+          tabBarIcon: ({color}) => (
+            <FontAwesome5Icon name="address-book" color={color} size={33} />
+          ),
+        }}
+      />
+
+      <Tab.Screen
+        name="Profile"
+        component={Profile}
+        options={{
+          tabBarLabel: 'Profile',
+          tabBarIcon: ({color}) => (
+            <FontAwesome5Icon name="user-circle" color={color} size={33} />
```

Comment:
I've changed this to `<FontAwesome5Icon name="user-tie" color={color} size={33} />`, you can try `user` or `user-alt` and choose one, I think they're better than user circle because the other icons don't have a circle.

Also, I think the size 30 for all the icons is better than 33.

Reply:
There won't be a profile tab, we said we'll make it link to the user's profile on github, please change it to sth like this ` onPress={() => Linking.openURL('https://github.coms')}>`, use a place holder profile until we have some backend to display the logged-in user profile.